central/xlsx/reader.go

```diff
@@ -196,15 +197,21 @@ func convertDeviceFields(rowElement *reflect.Value, xlsxRow []string, headerCol
 			if fieldValue != "" {
 				// get the Path defined in the MappingTable
 				if mapping, ok := fieldMappings[headerName]; ok && mapping.path != "" {
-					path := mapping.path
-					fieldPrefix := strings.SplitN(path, mappingPathSeparator, 2)[0]
+					splitPaths := strings.SplitN(mapping.path, mappingPathSeparator, 2)
+					fieldPrefix := strings.TrimSpace(splitPaths[0])
+					fieldName := strings.TrimSpace(splitPaths[1])
```

Comment:
```suggestion
					fieldName := ""
					if len(splitPaths > 1) {
						fieldName = strings.TrimSpace(splitPaths[1])
					}
```

SplitN doesn't gurantee there will be 2 elements
It may cause panic if the element is only one